src/index.js

```diff
@@ -68,118 +63,143 @@ query ($id: ID!) {
-          switch (event.__typename) {
-            case "ConnectedEvent":
-              prs.add(event.subject.url);
-              break;
-            case "CrossReferencedEvent":
-              prs.add(event.source.url);
-              break;
-          }
-        }
+/**
+ * Parse the response of spring backlog query.
+ * @param {Array} members - array of object contains members list
+ * @param {Array} response - response of query as array of object
+ * @returns {Array} - array of object which contains members with task
+ */
+function parseQuery (members, response) {
+  const data = response.map((items) => {
+    if (items.state === 'NOTE_ONLY') {
+      return (items.note)
+    } else if (items.state === 'CONTENT_ONLY') {
+      if (items.content.__typename === 'PullRequest') { return ('@' + items.content.author.login + ' ' + items.content.url) }
+      if (items.content.__typename === 'Issue') {
+        const people = items.content.assignees.nodes.map((item) => {
+          return '@' + item.login + ' '
+        })
+        return (people + ' ' + items.content.url)
       }
-    } else if (card.__typename === "PullRequest") {
-      prs.add(card.url);
     }
+    return ''
+  })
+  let processed = [...data]
+  for (let i = 0; i < members.length; i++) {
+    processed = processed.map((x) => x.replace(new RegExp('@' + members[i].name, 'g'), ''))
   }
-  return [...prs];
-}
-
-function prsToText(prs) {
-  return (
-    "Requests to review 😊\n" +
-    MENTION.split(",")
-      .map((x) => x.trim())
-      .map((x) => "@" + x)
-      .join(" ") +
-    "\n\n" +
-    prs.join("\n")
-  );
-}
-
-async function notify(data) {
-  return axios({
-    method: "POST",
-    url: NOTIFIER_URL,
-    data: "message=" + encodeURIComponent(data)
+  processed = processed.map((x) => x.replace(/(\r\n|\n|\r)/gm, ''))
+  data.forEach((items, index) => {
+    for (let i = 0; i < members.length; i++) {
+      if (items.includes('@' + members[i].name)) {
+        members[i].tasks.push(processed[index])
+      }
+    }
   })
+  return members
 }
 
-async function sendOpenPrs() {
-  const query = await octokit.graphql(CARDS_QUERY, { id: COLUMN_NODE_ID });
-  console.log(query);
-  console.log(query.node.cards.nodes);
-  const prs = extractPullRequests(query.node.cards.nodes);
-  console.log(prs);
-
-  const resp = await axios({
-    method: "POST",
-    url: NOTIFIER_URL,
-    data: "message=" + encodeURIComponent(prsToText(prs)),
-  });
-  console.log(resp.status);
-  console.log(resp.data);
+/**
+ * Request the GraphQL API of Github with SPRING_BACKLOG_CARDS_QUERY query
+ * @param {Array} members - array of object contains members list
+ */
+function backlogCardQuery (members) {
+  return octokit
+    .graphql(SPRING_BACKLOG_CARDS_QUERY, { id: COLUMN_NODE_ID })
+    .then((query) => {
+      return parseQuery(members, query.node.cards.nodes)
+    })
+    .catch((err) => {
+      console.error("Can't make test request for cards\n", err)
+      process.exit(1)
```

Comment:
Do we need it here? I guess we should catch errors in main function

Reply:
ya, we can.